README.md

```diff
@@ -1,2 +1,140 @@
-# flat-image-tree
-Spec for the Flat Image Tree (FIT) specification
+# Document master project for the Devicetree Specification #
+
+The latest release of the specification can be found at http://devicetree.org/specifications/ or https://github.com/devicetree-org/devicetree-specification/releases
+
+This [repository](https://github.com/devicetree-org/devicetree-specification) holds the source for the generation of the Devicetree Specification using Sphinx and LaTeX. 
+
```

Comment:
let me check with @walterchris if we can set up a mailing list on this under OSFF.